docker/dockerfiles/catalog

```diff
@@ -0,0 +1,11 @@
+FROM postgres:10
+
+LABEL maintainer="wesley.monte@ccc.ufcg.edu.br"
+
+ENV POSTGRES_USER=admin
+ENV POSTGRES_PASSWORD=admin
+ENV POSTGRES_DB=saps
```

Comment:
these data are sensitive. Maybe you should create a script that reads the data from a conf file and fulfill fields/create env variables wherever this data is needed. One option is: You can pass the variables as parameters when u run the container...

Reply:
not sure about that. But, we can replace them by placeholders, e.g. $POSTGRES_USER, anyway.

Reply:
These variables exist by default in postgres. Defining them like this I defined a standard user for the catalog. They can be changed at container startup time.